### PR TITLE
feat: add types package

### PR DESCRIPTION
**Pull request by davidebianchi**
Add new package to the monorepo, which contains the Console types

### CODE REVIEW

File: packages/console-sdk-microfrontend/src/MicrofrontendIntegrator.test.ts [hunk 1]
```diff
@@ -42,8 +42,17 @@ const consoleProps: IConsoleProps = {
     forceConfigUpdateChecksum: '',
     microfrontendPluginConfig: {},
 
-    selectedEnvironment: {},
-    selectedProject: {},
+    selectedEnvironment: {
+      envId: 'testEnvId',
+    },
```
Comment (davidebianchi) [inline packages/console-sdk-microfrontend/src/MicrofrontendIntegrator.test.ts:47]: This flag is typed different from the actual

**Review by pmarchini — APPROVED** [review]

File: packages/console-sdk-microfrontend/src/MicrofrontendIntegrator.test.ts [hunk 2]
```diff
@@ -42,8 +42,15 @@ const consoleProps: IConsoleProps = {
     forceConfigUpdateChecksum: '',
     microfrontendPluginConfig: {},
 
-    selectedEnvironment: {},
-    selectedProject: {},
+    selectedEnvironment: 'testEnvId',
```
Comment (fredmaggiowski) [inline packages/console-sdk-microfrontend/src/MicrofrontendIntegrator.test.ts:45]: this ought to change with #10

File: packages/console-sdk-microfrontend/src/types.ts [hunk 3]
```diff
@@ -61,14 +60,14 @@ export type IContexts = {
...
-  configMaps: ServiceConfigMap
+  configMaps: ConfigMaps
   services: Services
   unsecretedVariables: PublicVariable[]
 
   forceConfigUpdateChecksum: string
   microfrontendPluginConfig: Record<string, unknown>
 
-  selectedEnvironment: IEnvironment
+  selectedEnvironment: string
```
Comment (fredmaggiowski) [inline packages/console-sdk-microfrontend/src/types.ts:70]: Same, should change with #10 10

File: packages/console-types/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,856 @@
+# CHANGELOG
```
Comment (fredmaggiowski) [inline packages/console-types/CHANGELOG.md:1]: Are we using changelogs?

we should definitely setup the releaser workflow!

**Review by fredmaggiowski — APPROVED** [review]
LGTM

File: packages/console-types/CHANGELOG.md (hunk 4 above)
Comment (fredmaggiowski) [reply packages/console-types/CHANGELOG.md:1]: opened #11

**Pull request merged**